StreamChatClient_v3/APIClient/Endpoints/Payloads/ChannelCreatePayload_Tests.swift

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatClient_v3
+import XCTest
+
+class ChannelCreatePayload_Tests: XCTestCase {
+    func test_channelCreatePayloadInitPropogates() {
+        let cid = ChannelId.unique
+        let name = "testChannelName"
+        let imageURL = URL(string: "https://getstream.io/random_svg/?id=broken-waterfall-5&amp;name=Broken+waterfall")!
+        let team = "testTeam"
+        let members: Set<String> = [UserId.unique]
+        let invites: Set<String> = [UserId.unique]
+        let extraData: NameAndImageExtraData = .init(name: "test", imageURL: imageURL)
+
+        // Create ChannelCreatePayload
+        let payload = ChannelCreatePayload<DefaultDataTypes>(cid: cid,
+                                                             name: name,
+                                                             imageURL: imageURL,
+                                                             team: team,
+                                                             members: members,
+                                                             invites: invites,
+                                                             extraData: extraData)
+
+        // Assert ChannelCreatePayload inited correctly
+        XCTAssertEqual(payload.cid, cid)
+        XCTAssertEqual(payload.name, name)
+        XCTAssertEqual(payload.imageURL, imageURL)
+        XCTAssertEqual(payload.team, team)
+        XCTAssertEqual(payload.members, members)
+        XCTAssertEqual(payload.invites, invites)
+        XCTAssertEqual(payload.extraData, extraData)
+    }
+
+    func test_channelCreatePayload_encodedCorrectly() throws {
+        let payload = ChannelCreatePayload.test
+        let encodedJSON = try JSONEncoder.default.encode(payload)
+        let encodedString = String(data: encodedJSON, encoding: .utf8)!
+
+        // Assert ChannelCreatePayload encoded correctly
+        XCTAssertEqual(encodedString,
+                       "{\"invites\":[\"testUserId\"],\"name\":\"testName\",\"team\":\"testChannelTeam\",\"members\":[\"testUserId\"]}")
```

Comment:
I'm not sure how stable this is, since the order of the values in JSON is not stable. You can use our `AssertJSONEqual` helper for this. In the end, serializing the values to JSON is the only and main functionality this struct has.